### ISSUE TITLE
Support AIX ppc64

### ISSUE THREAD

**Issue by wiggin15** [python-greenlet/greenlet#159]
Looking at the `platform/switch_*` files, I see there is support for `ppc_aix`, `ppc_linux`, and `ppc64_linux`, but no support for `ppc64_aix`. Trying to use this platform, I am getting strange behavior - e.g. the examples from the docs get stuck, and in more complex usage (which I could not yet find a minimal working case for) I am getting an error similar to the one mentioned in #145. Maybe this issue is the same as #145, except it seems that the effort in that ticket is aimed specifically for IBM i and not AIX, and from what I can understand, the comments there don't seem to address the need to add support for a new platform.

It looks like copying `switch_ppc64_linux.h` -> `switch_ppc64_aix.h` and adding the following lines to `slp_platformselect.h` may work:
```C
#elif defined(__GNUC__) && defined(__powerpc64__) && defined(_AIX)
#include "platform/switch_ppc64_aix.h" /* gcc on AIX/PowerPC 64-bit */
```

**Comment by jamadden** [comment]
PRs are welcome!

**Comment by jamadden** [comment]
Fixed in #197.
